packages/formbuilder/src/Repeats.js

```diff
@@ -41,6 +41,7 @@ const createFormForItem = (
   const mappedFields = fieldsForForm.map(field => ({
     ...field,
     id: `${field.id}_${targetIndex}_FIELDS`
+    name: field.name || field.id // AkField requires name property or fails. 
```

Comment:
The formBuilder form validation still hadn't worked well and I've put this as a workaround to avoid failing while the dynamic form building. Ideally, validation should prevent from sending field without name prop...